test_setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# this one is to get everything set up without a hitch so you can run your
```

Comment:
What is "one"? Make your comments more formal and informative. They are for others to read more than yourself.

Reply:
I updated the script description but the rest of the comments do need some updating still.